### DISCUSSION TITLE
Additional math function: get bulge just from arc chord and angle

### DISCUSSION THREAD

**Discussion by nopria** [mozman/ezdxf#758] [Ideas]
`ezdxf.math.arc_to_bulge` needs arc center, radius, start angle and end angle to return bulge together with start and end points of arc, but sometimes you don't have all those information and you might want to know the bulge value just from start and end points of arc and its total angle. In such cases you can obtain the bulge value just from the chord and total angle of arc throught the following function (hoping I did the right math):

```
def bulge_r_c(r,c): # bulge from arc radius and chord
    x = c/(2*r)
    return x/(1+math.sqrt(1-x**2))
```

**Comment by mozman** [comment]
Thank you! Added the function with commit: 12f534ae

**Comment by nopria** [comment]
You might want to add also a function computing bulge from the arc spanning angle, which is surprisingly simple:

```
def bulge_from_arc_angle(angle): # angle in radians
    return math.tan(angle/4)
```

**Reply by mozman** [reply]
Thank you!
